### DISCUSSION TITLE
Show Templates with Caching Enabled

### DISCUSSION THREAD

**Discussion by ic360-com** [ExpressionEngine/ExpressionEngine#4578] [Feature Requests]
### Suggest an idea for ExpressionEngine

In the CP on the Template listings... it would be very useful to have a column (or another icon) showing which Templates have Caching switched on.
Maybe move these PHP and Caching icons into their own column?

**Comment by intoeetive** [comment]
https://github.com/ExpressionEngine/ExpressionEngine/pull/4616
